backend/notifications/views.py

```
@@ -57,3 +62,32 @@ def mark_read(self, request, pk=None):
     def mark_all_read(self, request):
         updated = services.mark_all_read(request.user)
         return Response({'updated': updated})
+
+
+class WhatsNewAnnouncementViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    serializer_class = WhatsNewAnnouncementSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)